### PR TITLE
DSCS-221

### PR DESCRIPTION
**Pull request by yetti**
@MingtaoSun @bsingh-nla 

Renamed `hasCopyType()` to `hasCopyRole()`. I thought it would be less likely to get confused with the `getCopyType()` method on the `Copy` object.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
   <groupId>au.gov.nla</groupId>
   <artifactId>amberdb</artifactId>
-  <version>1.2.6-SNAPSHOT</version>
+  <version>1.2.7-SNAPSHOT</version>
```
Comment (MingtaoSun) [inline pom.xml]: I thought we are not doing it any more for minor changes

Comment (yetti) [inline pom.xml]: I wasn't told that...

Comment (bsingh-nla) [inline pom.xml]: Based on the discussion I had with Shaun, if new change breaks existing code, we should update version number.

Comment (MingtaoSun) [inline pom.xml]: OK, sounds good

**Pull request merged**